js/package.json

```diff
@@ -18,7 +18,7 @@
     "watch": "webpack --watch"
   },
   "jupyterlab": {
-    "extension": "src/labplugin"
+    "extension": "src/lab_plugin"
```

Comment:
The file was renamed to `lab_extension` not `lab_plugin`. Am I missing something?

Reply:
No, I was ;).  I fixed it in a follow up.